### PR TITLE
feat: adjust og image for gallery detail

### PR DESCRIPTION
**Pull request by preschian**
- Fix https://github.com/kodadot/nft-gallery/issues/7997

image

### CODE REVIEW

File: ogi/components/OgImage/gallery.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+const props = defineProps<{
+  title: string;
+  image: string;
+  usd: string;
+  price: string;
+  network: string;
+}>();
+
+const cover: ComputedRef<CSSProperties> = computed(() => {
```
Comment (roiLeo) [inline ogi/components/OgImage/gallery.vue:17]: doesn't need computed, you can bind dirrectly the styles

File: ogi/components/OgImage/gallery.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+const parseUsd = computed(() =>
+  parseFloat(props.usd) ? `$${props.usd}` : '--',
+);
+</script>
+
+<template>
+  <img :src="image" :alt="title" :style="cover" class="h-full w-full" />
+
+  <div
+    class="flex flex-col justify-end h-full w-full bg-slate-900/85 text-white p-20 text-2xl font-bold absolute inset-0"
```
Comment (roiLeo) [inline ogi/components/OgImage/gallery.vue:33]: let's unify padding (https://github.com/kodadot/workers/pull/198/files) your solution might be better

**Review by roiLeo — APPROVED** [review]
otherwise lgtm

**Comment by preschian** [comment]
> Why do you need gql? is something in uniquery missing for you?

oh, my bad, I'm getting mixed with stats on the collection. will check

**Comment by preschian** [comment]
updated

| price | empty |
|--------|--------|
| image | image |

**Review by roiLeo — APPROVED** [review]

**Review by yangwao — APPROVED** [review]

**Comment by yangwao** [comment]
Can we then deploy this for production as well? cc @preschian 
Thank you!

**Comment by preschian** [comment]
I've got an unexpected error 🤣

doesn't work on kodadot https://ogi.kodadot.workers.dev/ahk/gallery/134-5
it works fine on my workers https://ogi.preschian-cdn.workers.dev/ahk/gallery/134-5

same code

still checking

**Comment by roiLeo** [comment]
```log
[Vue Router warn]: uncaught error during route navigation:
[[vite-node] [ERR_LOAD_URL] @polkadot/util
at @polkadot/util] {
  cause: {
    statusMessage: 'Vite Error',
    message: '[vite-node] [ERR_LOAD_URL] @polkadot/util',
    stack: '[vite-node] [ERR_LOAD_URL] @polkadot/util\nat @polkadot/util'
  },
  statusCode: 500,
  fatal: false,
  unhandled: false,
  statusMessage: 'Vite Error',
  data: undefined,
  __nuxt_error: true
}
```

**Pull request merged**